benchmark/benchmark_runner.py

```diff
@@ -242,7 +242,8 @@ def run_kuzu(serialized_graph_path):
         process = subprocess.Popen(
             tuple(benchmark_cmd), stdout=subprocess.PIPE)
         for line in iter(process.stdout.readline, b''):
-            print(line.decode("utf-8"), end='')
+            sys.stdout.write(line.decode("utf-8"))
+            sys.stdout.flush()
```

Comment:
You can also use the [flush argument of print](https://docs.python.org/3/library/functions.html#print) to explicitly flush, which is a little more concise.

Reply:
good idea